roles/ngi_pipeline/templates/irma_ngi_config.yaml.j2

```diff
@@ -85,3 +85,5 @@ paths: # Hard code paths here if you are that kind of a person
 logging: 
     log_file: "{{ ngi_pipeline_logs }}/ngi_pipeline.log" 
 
+mail:
+    recipient: ngi_pipeline_operators@scilifelab.se
```

Comment:
Not strictly necessary, but if you think we want to change this value easier in the future, or have separate versions for Stockholm and Uppsala, then adding this as a variable from either `ngi_pipeline/defaults/` or `host_vars` might be suitable.